notes/returned-mail-received.vbs

```diff
@@ -376,51 +410,107 @@ IF CRF_checkbox = CHECKED THEN pending_verifs = pending_verifs & "Change Report
 pending_verifs = trim(pending_verifs) 	'takes the last comma off of pending_verifs
 IF right(pending_verifs, 1) = "," THEN pending_verifs = left(pending_verifs, len(pending_verifs) - 1)
 '----------------------------------------------------------------------------------------------------TIKL
+due_date = dateadd("d", 10, date_requested)	'defaults the due date for all verifications at 10 days
 'Call create_TIKL(TIKL_text, num_of_days, date_to_start, ten_day_adjust, TIKL_note_text)
-IF ADDR_actions <> "no response recieved" THEN Call create_TIKL("Returned mail rec'd contact from the client should have occurred regarding address change. If no response-verbal or written, please take appropriate action.", 10, date, True, TIKL_note_text)
+IF ADDR_actions <> "no response received" THEN Call create_TIKL("Returned mail rec'd contact from the client should have occurred regarding address change. If no response-verbal or written, please take appropriate action.", 10, date, True, TIKL_note_text)
 'starts a blank case note
+'----------------------------------------------------------------------------------------------------CASENOTE
 CALL start_a_blank_case_note
 CALL write_variable_in_CASE_NOTE("Returned mail received " & ADDR_actions)
 CALL write_bullet_and_variable_in_CASE_NOTE("Received on", date_received)
- 'Address Detail
- IF mailing_address_confirmed = "YES" THEN
- 	' Call write_variable_in_CASE_NOTE("* The address on ADDR was reviewed and is correct.")
+CALL write_bullet_and_variable_in_CASE_NOTE("Client contacted", client_contact_dropdown)
+IF mailing_address_confirmed = "YES" THEN  'Address Detail
 	CALL write_variable_in_CASE_NOTE("* Returned mail received from: " & mail_line_one)
 	If mail_line_two <> "" Then CALL write_variable_in_CASE_NOTE("                             " & mail_line_two)
-	CALL write_variable_in_CASE_NOTE("                             " & mail_city_line & ", " & mail_state_line & " " &   mail_zip_line)
+	CALL write_variable_in_CASE_NOTE("                               " & mail_city_line & ", " & mail_state_line & " " &   mail_zip_line)
 END IF
 IF residential_address_confirmed = "YES" THEN
 	CALL write_variable_in_CASE_NOTE("* Returned mail received from: " & resi_addr_line_one)
 	If resi_addr_line_two <> "" Then CALL write_variable_in_CASE_NOTE("                               " & resi_addr_line_two)
-	CALL write_variable_in_CASE_NOTE("                               " & resi_addr_city & ", " & resi_addr_state & " " & resi_addr_zip)
+	CALL write_variable_in_CASE_NOTE("         		                      " & resi_addr_city & ", " & resi_addr_state & " " & resi_addr_zip)
+	CALL write_bullet_and_variable_in_case_note("Verification(s) received", pending_verifs)
```

Comment:
I don't know why this would be here - you might end up with the information in the NOTE twice and I thought these checkboxes referenced verifications that were requested.

Reply:
Verification requested and verification received should be handled as two separate processes here